styles/globals.css

```diff
@@ -80,6 +80,31 @@ a {
   }
 }
 
+/* Fade-in animation on scroll */
+.appear {
+  transition: all 1s;
+  opacity: 0;
+  transform: translateY(80px);
+}
+
+.appear.inview {
+  opacity: 1;
+  transform: none;
+  transition-delay: 0.3s;
+}
+
+/* Fade-in background animation on scroll */
+.fade_bg {
+  opacity: 0.4;
+  /* transform: translate(0, 20px); */
+  transition: all 2s;
+}
+
+.fade_bg.show {
+  opacity: 1;
+  /* transform: translate(0, 0); */
+}
```

Comment:
it looks like you're doing all animations in CSS right?

that's fine, just want to make you aware we also have Framer Motion for JS-based animations.

Your call 🌈 

docs: https://www.framer.com/motion/